payments-core/src/main/java/com/stripe/android/view/CardInputWidget.kt

```diff
@@ -592,48 +593,6 @@ class CardInputWidget @JvmOverloads constructor(
         if (state is Bundle) {
             postalCodeEnabled = state.getBoolean(STATE_POSTAL_CODE_ENABLED, true)
             isShowingFullCard = state.getBoolean(STATE_CARD_VIEWED, true)
-            updateSpaceSizes(isShowingFullCard)
-            placement.totalLengthInPixels = frameWidth
-            val cardStartMargin: Int
-            val dateStartMargin: Int
-            val cvcStartMargin: Int
-            val postalCodeStartMargin: Int
-            if (isShowingFullCard) {
-                cardStartMargin = 0
-                dateStartMargin = placement.getDateStartMargin(isFullCard = true)
-                cvcStartMargin = placement.getCvcStartMargin(isFullCard = true)
-                postalCodeStartMargin = placement.getPostalCodeStartMargin(isFullCard = true)
-            } else {
-                cardStartMargin = -1 * placement.hiddenCardWidth
-                dateStartMargin = placement.getDateStartMargin(isFullCard = false)
-                cvcStartMargin = placement.getCvcStartMargin(isFullCard = false)
-                postalCodeStartMargin = if (postalCodeEnabled) {
-                    placement.getPostalCodeStartMargin(isFullCard = false)
-                } else {
-                    placement.totalLengthInPixels
-                }
-            }
-
-            updateFieldLayout(
-                view = cardNumberTextInputLayout,
-                newWidth = placement.cardWidth,
-                newMarginStart = cardStartMargin
-            )
-            updateFieldLayout(
-                view = expiryDateTextInputLayout,
-                newWidth = placement.dateWidth,
-                newMarginStart = dateStartMargin
-            )
-            updateFieldLayout(
-                view = cvcNumberTextInputLayout,
-                newWidth = placement.cvcWidth,
-                newMarginStart = cvcStartMargin
-            )
-            updateFieldLayout(
-                view = postalCodeTextInputLayout,
-                newWidth = placement.postalCodeWidth,
-                newMarginStart = postalCodeStartMargin
-            )
```

Comment:
We shouldn't be doing UI operations when restoring state as the `width` is not known yet.